.github/actions/run-mf-tests/action.yaml

```diff
@@ -23,7 +30,8 @@ runs:
       python-version: "${{ inputs.python-version }}"
   - name: Run Tests
     shell: bash
-    run: make test
+    run: make -e PARALLELISM=${{ inputs.parallelism }} ${{ inputs.make-target }}
     env:
-      MF_SQL_ENGINE_URL: ${{ secrets.MF_REDSHIFT_URL }}
-      MF_SQL_ENGINE_PASSWORD: ${{ secrets.MF_REDSHIFT_PWD }}
\ No newline at end of file
+      MF_SQL_ENGINE_URL: ${{ inputs.mf_sql_engine_url }}
+      MF_SQL_ENGINE_PASSWORD: ${{ inputs.mf_sql_engine_password }}
+      METRICFLOW_CLIENT_EMAIL: ci-tester@gmail.com
```

Comment:
Wait, what? Is this a company-managed account?

Reply:
That is a bit strange. @WilliamDee know about this one?